### ISSUE TITLE
Shorts in the video page

### ISSUE THREAD

**Issue by Jinsgo** [ParticleCore/Iridium#981]
Hello. Congratulations on your extension, it's working really well!

I noticed that the site had a section dedicated to shorts. I was wondering if it was also possible to remove the shorts suggestions from the video page (framed in red on the screenshot). 
![Capture d'écran 2024-04-30 125209](https://github.com/ParticleCore/Iridium/assets/69437465/1f3b5a83-191f-4cc4-94ab-ee83f467ed95)

It seems that the code that manages this carousel is this:
![Capture d'écran 2024-04-30 125628](https://github.com/ParticleCore/Iridium/assets/69437465/9b445a33-1ddf-47c4-99ad-f9830a7e21f8)

Thanks for your work!

**Comment by ParticleCore** [comment]
Hi, sure thing, I actually planned on doing that from the start and completely forgot. Will address it in the next version update.

**Comment by Jinsgo** [comment]
Great! Thanks again 👍

**Comment by ParticleCore** [comment]
Will tie this up after #994 because I want to fix things up before adding more

**Comment by ParticleCore** [comment]
Feature seems to be working well, I've bundled it with the other settings that already existed, but was unsure if it made more sense to put it in the video page settings, but for now it will stay like this.

Here's an example showing the video page with the setting enabled (show shorts), disabled (don't show shorts), and then enabled again. Each time the setting was changed, the page was refreshed.

![1](https://github.com/user-attachments/assets/528eb880-7a4a-42eb-9b65-5db8a976a03c)
